TACC/Core-Styles
# feat(settings+elements): docs font

## Overview / Changes

For https://github.com/TACC/TACC-Docs:
- Add font settings.
- Set font size of main content.

## Related

- [TUP-297](https://jira.tacc.utexas.edu/browse/TUP-297)
- https://github.com/TACC/TACC-Docs/pull/2

## Testing

0. Load https://github.com/TACC/TACC-Docs/pull/2 at [dc9a770](https://github.com/TACC/TACC-Docs/pull/2/commits/dc9a7705c3d22d3e312ca557a2330ffc4fb656dc).
1. Verify font of main body (not sidebar) is 16px.

## UI

![docs font size](https://user-images.githubusercontent.com/62723358/214080069-df735763-c7b7-46e7-ab6c-71843dff788f.png)
